### ISSUE TITLE
Add a footer section 

### ISSUE THREAD

**Issue by shashank-iter** [Web3-Bharat-ITER/UI-Component-Library#18]
The footer section must contain a logo, quick links, address and contact details
And copyright text at the bottom. You can take design inspiration from any website and add some creativity to it. Do properly read the contribution guidelines and attach a screenshot of your work with the pull request. Happy Hacktober Month.

**Comment by Shridhar-dev** [comment]
I am interested in working on the footer section, can you assign it to me?

**Comment by shashank-iter** [comment]
Yeah sure @Shridhar-dev, you can go ahead with this. Make sure you read the contribution guidelines and add a screenshot of your work while making a PR.

**Comment by Shridhar-dev** [comment]
yeah sure!

**Comment by hafidzzakky** [comment]
hey @shashank-iter can i join to this project too ?

**Comment by kasib12** [comment]
@shashank-iter @Jyo561Python I would like to work on this issue. I am sure I can provide much needed designs to the footer section. Please assign the issue to me.

**Comment by shashank-iter** [comment]
You can contribute to this issue.

**Comment by kasib12** [comment]
Thanks @shashank-iter. I already prepared it before-hand.

**Comment by shashank-iter** [comment]
@kasib12 great. Make a PR.

**Comment by kasib12** [comment]
@shashank-iter already did :)
